build.sbt

```diff
@@ -55,6 +60,8 @@ val timeBenchmark =
     .enablePlugins(JmhPlugin)
     .settings(commonSettings: _*)
     .settings(
+      // FIXME: does not compile with dotty
+      crossScalaVersions := List(scalaVersion.value),
```

Comment:
I think the ambiguous implicits are expected. We should explicitly insert these parameters.

For the Java parsing errors, I guess this is a bug in Dotty?